src/index.css

```diff
@@ -99,6 +105,48 @@ body {
   color: var(--color);
 }
 
+.main-footer {
+  padding-left: 4rem;
+  position: relative;
+  bottom: 0;
+  width: 100%;
+  text-align: center;
+  color: #15191d;
+  background-color: #4facfe;
+  float: left;
+}
+
+.footer-bottom {
+  background-color: #00f2fe;
+  width: 100%;
+  height: 2.5rem;
+  padding-bottom: 0px;
+  padding-top: 0px;
+  float: left;
+}
+
+.page-wrapper {
+  min-height: 100%;
+}
+
+.col {
+  width: 21%;
+  padding-top: 5px;
+  float: left;
+  font-size: var(--small-text);
+  font-family: var(--lato);
+  text-align: left;
+}
+
+.footer-container {
+  float: left;
+  width: 100%;
+}
+
+.list-unstyled {
+  list-style: none;
+}
+
 .large-btn {
   outline: none;
   border: 1px solid var(--bg-fallback);
```

Comment:
kindly please move css styles for the footer to the `App.css`.  where all other styles for features are.